h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -1595,6 +1597,7 @@ public CSVStreamParams setSeparator(byte separator) {
     Chunk[] _curChks;
     int _lastChkIdx;
     public volatile int _curChkIdx; // used only for progress reporting
+    private final Map<Integer, String[]> escapedCategoricalLevels = new HashMap<>();
```

Comment:
incosistent naming

Reply:
also I think this could be more defensive and this Map could be Immutable and as such should be a return value from escapeCategoricalVecDomains and initialized in constructor

Reply:
Aligned the naming to the corresponding method. Also made the method return immutable set. Thank you.

Reply:
i ment the convention to use _ as prefix for instance vars, I dont like it and for new classes I dont honor it, but for old classes I think we should keep them consitent and either remove all _ or add new fields with _

Reply:
Have the same approach personally. I just forgot to add it in this case. Done, it's there.